<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemedicineClass</class>
 <widget class="QMainWindow" name="TelemedicineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telemedicine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>xx市xx医院远程诊断系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">空中花园</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>六合区</string>
     </property>
     <item>
      <property name="text">
       <string>内科</string>
      </property>
      <item>
       <property name="text">
        <string>内分泌科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>呼吸科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>消化科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>妇科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>神经内科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>肾内科</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>外科</string>
      </property>
      <item>
       <property name="text">
        <string>肝胆外科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>血管外科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>心胸外科</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>耳鼻咽喉科</string>
       </property>
      </item>
     </item>
    </item>
    <item>
     <property name="text">
      <string>浦口区</string>
     </property>
     <item>
      <property name="text">
       <string>妇产科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>老年医学科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>儿科</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>溧水区</string>
     </property>
     <item>
      <property name="text">
       <string>骨科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>血管外科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>心脏科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>脑科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>神经内科</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>高淳区</string>
     </property>
     <item>
      <property name="text">
       <string>呼吸科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>消化科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>泌尿外科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中医科</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QLabel" name="CT_Img_Label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CT影像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="yearLcdNumber">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="value" stdset="0">
     <double>1900.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="mounthLcdNumber">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="dayLcdNumber">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>16</width>
      <height>391</height>
     </rect>
    </property>
    <property name="value">
     <number>30</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开 始 诊 断</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>信息</string>
     </attribute>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>患者名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="maleRadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="femaleRadioButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>女</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="ageSpinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>18</number>
      </property>
     </widget>
     <widget class="QComboBox" name="ethniComboBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>汉</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>民族：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ssnLineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>医保卡编号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="photoLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Panel</enum>
      </property>
      <property name="text">
       <string>暂无照片</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>病历</string>
     </attribute>
     <widget class="QTextEdit" name="caseTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>181</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled/>
       </palette>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="basicTableView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>601</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Telemedicine.qrc"/>
 </resources>
 <connections/>
</ui>
